setup.sh

```diff
@@ -103,14 +103,14 @@ dbsetup(){
       ./create-mysql-db.sh
     fi
   else
-    echo 'Please install MYSQL Server'
+    echo 'Please install MYSQL Server.'
     exit
   fi
 }
 
 # Returns 1 if a global command-line program installed, else 0.
 # For example, echo "node: $(program_is_installed node)".
```

Comment:
Fixed in https://github.com/cfpb/cfgov-refresh/commit/858e4161759a370417351036e64fc95f31c218ab